docs/dev/openstack-howto.md

```diff
@@ -0,0 +1,57 @@
+# OpenStack HOWTO
```

Comment:
The libvirt docs live under `docs/dev` because they are never intended to be user-facing.  OpenStack, on the other hand, is intended to be user-facing and just hasn't graduated to non-experimental yet.  Can we put this in `docs/user/openshift/README.md` or some such and open with:

```markdown
# OpenStack

**Support for launching clusters on OpenStack is experimental.**
```

or some such?  Then we can just remove the caveat when we feel comfortable with it, and won't cause downstream issues with breaking links and such.

Using an `openstack` directory also gives us a place to drop an `OWNERS` file so we can one day return to having separate per-platform mantainers.

How does that sound?

Reply:
Sounds great.  I'll fix it up.

The pointer to how to set up a development environment is dev specific, so maybe I'll leave a small openstack dev doc for that.

There's also some more content from #909 I'll pull in.

Thanks for the quick feedback!